apps/prairielearn/src/pages/instructorAssessmentUploads/instructorAssessmentUploads.html.ts

```diff
@@ -0,0 +1,280 @@
+                      `}
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+function csvHelpInstanceQuestionScores() {
+  return html`
+    <p>
+      Upload a CSV file in the format of the
+      <code><i>&lt;assessment&gt;</i>_submissions_for_manual_grading.csv</code>
+      file from the Downloads page. See the
+      <a href="https://prairielearn.readthedocs.io/en/latest/manualGrading/"
+        >Manual Grading documentation</a
+      >.
+    </p>
+    <p>Alternatively, the CSV file can be in the format:</p>
+    <pre class="ml-4">
+uid,instance,qid,score_perc,feedback
+student1@example.com,1,addTwoNumbers,34.5,The second step was wrong
+student2@example.com,1,addTwoNumbers,78.92,
+student2@example.com,1,matrixMultiply,100,Great job!</pre
+    >
+    <p>
+      The <code>instance</code> column indicates which assessment instance to modify. The total
+      scores for the assessments will be automatically recalculated. The feedback will be attached
+      to the most recent student submission. To change the per-question points, replace the
+      <code>score_perc</code> column above with a <code>points</code> column.
+    </p>
+    <p>
+      By default the updates above change the manual portion of the grades for the instance
+      question, in such a way that the sum of the new manual portion with the existing autograding
+      portion matches the total listed in the <code>points</code> or <code>score_perc</code> columns
+      above. It is, however, possible to replace these columns with <code>manual_points</code> and
+      <code>auto_points</code> (or <code>manual_score_perc</code> and <code>auto_score_perc</code>)
+      for more fine-tuned change of grade components.
+    </p>
+  `;
+}
+
+function csvHelpAssessmentInstanceScores() {
+  return html`
+    <p>Upload a CSV file like this:</p>
+    <pre class="ml-4">
+uid,instance,score_perc
+student1@example.com,1,63.5
+student2@example.com,1,100</pre
+    >
+    <p>
+      The example above will change the total assessment percentage scores for
+      <code>student1@example.com</code> to 63.5% and for <code>student2@example.com</code> to 100%.
+      The <code>instance</code> column indicates which assessment instance to modify, and should be
+      <code>1</code> if there is only a single instance per student.
+    </p>
+    <p>
+      Alternatively, the total assessment points can be changed with a CSV containing a
+      <code>points</code> column, like:
+    </p>
+    <pre class="ml-4">
+uid,instance,points
+student1@example.com,1,120
+student2@example.com,1,130.27</pre
+    >
+    <p>For assessments using group work, use the <code>group_name</code> column instead:</p>
+    <pre class="ml-4">
+group_name,instance,score_perc
+myhappygroup,1,95
+greatgroup,1,85</pre
+    >
+  `;
+}
+
+function uploadInstanceQuestionScoresModal({ csrfToken }: { csrfToken: string }) {
```

Comment:
I don't remember what the convention was, but I believe these components have been using PascalCase, isn't that right?

Reply:
Yes, good catch. Updated all components here to `PascaleCase` in [ad361b5](https://github.com/PrairieLearn/PrairieLearn/pull/9777/commits/ad361b501b5eaaa21891a8a7c389492448d3d51f).